pipelines/models/tensorflow-facedetection/README.md

```diff
@@ -15,7 +15,7 @@ docker run -d -u $(id -u):$(id -g) --rm -v ${PWD}/models:/model:Z -p 9000:9000 o
 git clone https://github.com/openvinotoolkit/model_server.git
 cd model_server/demos/face_detection/python
 # Patch the requirements file, because tensorflow 2.11.0 is not working
-sed -i 's/2.11.0/2.13.0/' client_requirements.txt
+sed -i 's/tensorflow-serving-api==2.11.0/tensorflow-serving-api==2.13.0/' ../../common/python/requirements.txt
 python -m venv .venv
 source .venv/bin/activate
 pip install -r ../../common/python/requirements.txt
```

Comment:
I actually couldn't get this particular test request to work...I had issues with Python dependencies.